src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -50,36 +50,38 @@
     <th>Commit</th>
     <th>Link</th>
   </thead>
-  <tbody>
-    <tr *ngFor="let node of sortedBuilds"
-      [ngClass]="{'table-warning': !node.coherent}"
+  <tbody @noop>
+    <ng-container *ngFor="let node of sortedBuilds; trackBy getBuildId">
```

Comment:
Does this work? In my experience, things inside tbody tags that were not supposed to be there get messed up really bad.  Firefox will physically move the element outside the table...

Reply:
"ng-container" is a special angular element. it doesn't get rendered into the dom. Its specifically for this purpose.